taichi/ir/statements.h

```diff
@@ -1200,6 +1200,19 @@ class ClearListStmt : public Stmt {
 // Checks if the task represented by |stmt| contains a single ClearListStmt.
 bool is_clear_list_task(const OffloadedStmt *stmt);
 
+/**
+ * The statement corresponding to a set list up-to-date task.
```

Comment:
I think it's better to add more explanations here such as this statement does not modify the list itself and only sets the boolean variable |up_to_date|.

Reply:
Will do!